docs/using/common.md

```diff
@@ -215,13 +224,14 @@ If you do, keep in mind that features supported by the `start.sh` script and its
 
 The default Python 3.x [Conda environment](https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html) resides in `/opt/conda`.
 The `/opt/conda/bin` directory is part of the default `jovyan` user's `${PATH}`.
-That directory is also whitelisted for use in `sudo` commands by the `start.sh` script.
+That directory is also allow listed for use in `sudo` commands by the `start.sh` script.
```

Comment:
I think `allow listed` -> `allowed` would feel more natural.

Reply:
Or `explicitly allowed` maybe? Is there configuration that `directly explicitly allowed for use in sudo commands by the start.sh script`? I don't really grasp the technical meaning of this. Hmmm...

Is it that any executable files in that folder, has some `sudo` flag set, allowing users to run them as sudo? Eh, hmmm... Well, if anyone if you understands this technically, it would be great to have a clarification because I don't. But, otherwise +1 for something like `setup ahead of time to be allowed`.

Reply:
I think this refers to these lines:
https://github.com/jupyter/docker-stacks/blob/master/base-notebook/start.sh#L153-L154

But I have no idea what secure_path is.
@consideRatio I think you actually might know this :)

Reply:
It effectively controls the `PATH` variable seen by sudo, so that `sudo ls` searches for ls in `secure_path` and not the user's modified path which could result in `/my/path/to/mailicious/ls`

Reply:
Thanks @manics !

```
That directory is also searched for binaries when run using `sudo` (`sudo my_binary` will search for `my_binary` in `/opt/conda/bin/`
```

I think something like this would be useful and understandable by a lot of people.

Reply:
Thank you @manics!! :heart: :tada: